storage-inventory/ops/README.md

```diff
@@ -128,139 +179,318 @@ The [`CADC Direct Data Service`](https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/en
+            - the `org.opencadc.minoc.inventory.username` database account is the 'Inventory admin user' configured when creating the [database](#configuration-database)
+    - See the [opencadc storage inventory minoc](https://github.com/opencadc/storage-inventory/tree/master/minoc) documentation for more configuration details.
     - when configuring the storage adapter for minoc to use (see **Storage** above) be sure to test that the containers deployed on your system can access the provided storage.
     - test with, e.g., `curl https://www.example.org/minoc/availability`
 
-1. **luskan**
-
-    - Container image: Use the latest `storage-inventory/luskan` image from `images.opencadc.org`
-    - See the [opencadc storage inventory luskan](https://github.com/opencadc/storage-inventory/tree/master/luskan) documentation for configuarion details.
-        - the 'uws' account in the `catalina.properties` file is the `tapadm` account created when the database was configured above.
-        - the 'query' account in the `catalina.properties` file is the `tapuser` account.
-        - (Should probably be in the luskan docs)  For the `cadc-tap-tmp.properties` file:
-            - the `{server name}` in the `baseURL` is the public name of the proxy fronting your `luskan` service (e.g. `www.example.org`).
-            - the `{luskan path}` is the path to your `luskan service` (likely just 'luskan', e.g. `www.example.org/luskan`).
-            - the `{local directory}` in `baseStorageDir` the is the location _on the container_ which the `baseURL` `/results` path will be mapped to.
+1. <a id='configuration-luskan'>**luskan**</a> - Query service
+
+    - Required for: Storage site and Global site
+    - Container image: Use the latest `storage-inventory/luskan` image from [`images.opencadc.org`](#resource-repository)
+    - Configuration notes:
+        - `luskan.properties`:
+            - `org.opencadc.luskan.isStorageSite` - for a storage site, this should be set to `true`.  The content of the inventory database is different between a storage site and a global site.
+            - `org.opencadc.luskan.allowedGroup` is an IVOA GMS group [resourceID](#term-resourceid).
+                - the GMS service must be configured in the available Registry service.
+        - `catalina.properties`:
+            - the `org.opencadc.luskan.uws.username` database account is generally the same as the 'TAP admin user' configured when creating the [database](#configuration-database).
+            - the `org.opencadc.luskan.tapadm.username` database account is the same 'TAP admin user'.
+            - the `org.opencadc.luskan.query.username` database account is the 'TAP query user' account.
+        - `cadc-tap-tmp.properties`:
+            - `org.opencadc.tap.tmp.TempStorageManager.baseURL` is the URL for _this_ `luskan` service, plus the `/results` path.
```

Comment:
I don't understand the part about `/results` path.

Reply:
Clarified